README.rst

```diff
@@ -1,7 +1,7 @@
 WebOb
 =====
 
-.. image:: https://travis-ci.org/Pylons/webob.png?branch=master
+.. image:: https://travis-ci.com/Pylons/webob.png?branch=master
         :target: https://travis-ci.org/Pylons/webob
```

Comment:
Thanks for this PR. Could you please grep the code for other uses of travis-ci.org and update them as well. For example, line 5 in this file. :-)